arlas-core/src/main/java/io/arlas/server/impl/elastic/utils/ElasticClient.java

```diff
@@ -264,7 +263,7 @@ public void close() {
                 this.sniffer.close();
             }
             this.client.close();
```

Comment:
Don't see the code around, but shouldn't the `close()`be in a finally?

Reply:
sniffer.close does not raise any exception, while client.close() raises an IOException which is caught.